### DISCUSSION TITLE
Copy text from chat?

### DISCUSSION THREAD

**Discussion by cocucku13** [darrenburns/elia#13] [Ideas]
How can I copy the answer from the chat?

**Comment by Emasoft** [comment]
even a button for exporting to txt file would be great!

**Comment by darrenburns** [comment]
Agreed! I've created an issue to track this here: #10

**Comment by nowilton** [comment]
I would really like to see this function, can you tell me when it will be implemented?

**Reply by darrenburns** [reply]
Added in 1.0.0

**Comment by darrenburns** [comment]
I've added this in 1.0.0.

**Reply by jcvcastro** [reply]
Thanks a lot!

But, for some reason, did not work on my system. I am using Marjar/Arch linux, with alacritty terminal. When pressing y (or c), pops up a messaging saying that was copied, but it does not goes to my clipboard. Past does not work.

I need to have some app installed? Like xclip, for example? (p.s.: I have xclip installed).

**Reply by jcvcastro** [reply]
Updating:

Using kitty terminal, it worked perfectly!

**Reply by jcvcastro** [reply]
Update:
Alacritty is working as expected! The problem was tmux, not alacritty!

**Comment by darrenburns** [comment]
I've updated this to use Pyperclip, so it should be supported across more environments now, regardless of your terminal.
